### DISCUSSION TITLE
Add feature to "follow" other snakes!

### DISCUSSION THREAD

**Discussion by uncleBlobby** [BattlesnakeOfficial/feedback#119] [Feature Requests]
Right now, if you want to look at how another developer's snake is doing, you need to either:

1. enter their url manually (or bookmark it)
2. find them in the leaderboard and click their name
3. ????

Add a button to follow other snakes. Then when you log in to your own page, there's another panel where you can see some of the latest information for those you follow as well!

Maybe instead of following a particular snake, follow the developer?  It could be tied to the "cheer" mechanic, maybe -- that already has a button.

Is there some reason this doesn't already exist?  Thoughts?

**Comment by andrewdmaclean** [comment]
Definitely giving this one a ⬆️

**Reply by bvanvugt** [reply]
Same! Love it.
